Assignments/10 - Culminating/README.md

```diff
@@ -0,0 +1,96 @@
+**The culminating is worth 30% of your final mark.**
+
+Task(s)
+-------
+
+
+###Part 1
+
+The following survey will be conducted during class time to you and your ICS 3C and 3U peers. 
+
+  1. On a scale from 0 to 10, how fun is programming?
+  2. What percentage of the software development process would you estimate is spent writing code?
+  3. Name one aspect of being a good digital citizen.
+  4. Name one job that you can pursue with a background in computer science.
+  5. Briefly explain why programming is a useful skill.
+
+You will be given the results in the form of a spreadsheet. Using the knowledge you've acquired from this course, store the results in **two different ways**.
+
+Write Python programs to determine at least **five** of the following:
+
+* The median result in response to the first question.
+* The mode (i.e., most common) result in response to the first question.
+* The average result in response to the second question.
+* The most common word that is at least 6 letters long in response to the third question.
+* The three most common words in response to the fourth question.
+* The student(s) who wrote the highest total number of characters in their responses.
+* The student(s) who wrote the lowest total number of characters in their responses.
+
+
+###Part 2
+Write a reflection paper about your process in Part 1. In fewer than 200 words total, respond to **all** the following:
+
+  1. Which way of storing data allowed for easiest access? Why?
+  2. Which of the five tasks was the most challenging? Why?
+  3. Discuss some of the planning and debugging strategies you used to overcome challenges when writing your programs.
+
+
+###Part 3
+
+In fewer than 300 words total, respond to **all** of the following:
+
+  1. Which computer peripheral device do you use the most? Find three computer tasks that cannot be accomplished without this device. Why can't these three tasks be done without this device?
+  2. What are the stages in the software development life cycle "waterfall model"? Which stage do you think would take the most time in a software project? Why?
+  3. Name two negative effects of computer use on the environment. Select one of the negative effects and explain what you can do to reduce this effect.
```

Comment:
Can you add in something about not including printing paper. I was them to think outside the paper world.